massa-module-cache/src/controller.rs

```diff
@@ -120,6 +120,11 @@ impl ModuleCache {
         bytecode: &[u8],
         execution_gas: u64,
     ) -> Result<RuntimeModule, CacheError> {
+        if bytecode.len() > self.cfg.max_module_length as usize {
```

Comment:
I believe it would fit better in "load_module_info" that already manages all kinds of bytecode invalidity errors

Reply:
Done.